db/schema.rb

```diff
@@ -30,4 +31,5 @@
     t.datetime "updated_at", null: false
   end
 
+  add_foreign_key "posts", "users"
```

Comment:
delete posts?

Reply:
need both to establish the connection I think!